src/containers/HomePage.js

```diff
@@ -46,8 +69,9 @@ class HomePage extends React.Component {
             containerElement={<div className="maps-container" />}
             mapElement={<div className="google-maps" />}
             handleClick={this.mapClick}
-            target={!this.props.match.isExact ? this.state.target : null}
-            coords={this.state.coords}
+            target={(location.pathname == '/create-target') ? target : null}
```

Comment:
You have this route in a routesPaths file, right? Can you compare to that constant instead of typing it by hand?